tensordict/tensordict.py

```diff
@@ -7008,31 +7000,22 @@ def _change_batch_size(self, new_size: torch.Size) -> None:
             del self._orig_batch_size
         self._batch_size = new_size
 
-    def get(
-        self,
-        key: NestedKey,
-        default: str | CompatibleType = NO_DEFAULT,
-        _return_original_tensor: bool = False,
-    ) -> CompatibleType:
-        # TODO: temporary hack while SavedTensorDict and LazyStackedTensorDict don't
-        # support nested iteration
-        include_nested = not isinstance(self._source, (LazyStackedTensorDict,))
-
-        if key in self._source.keys(include_nested=include_nested):
-            item = self._source.get(key)
-            transformed_tensor = getattr(item, self.custom_op)(
-                **self._update_custom_op_kwargs(item)
-            )
-            if not _return_original_tensor:
-                return transformed_tensor
-            return transformed_tensor, item
-        else:
-            if _return_original_tensor:
-                raise RuntimeError(
-                    "_return_original_tensor not compatible with get(..., "
-                    "default=smth)"
-                )
-            return self._default_get(key, default)
+    def _get_str(self, key, default):
+        tensor = self._source._get_str(key, default)
+        if tensor is default:
+            return tensor
+        print("tensor", tensor, key, default, self._source)
+        return self._transform_value(tensor)
+
+    def _get_tuple(self, key, default):
+        tensor = self._source._get_tuple(key, default)
+        if tensor is default:
+            return tensor
+        print("tensor", tensor, key, default, self._source)
```

Comment:
@vmoens These prints may have been forgotten (?)

Reply:
Yep sorry about that
I have to fix a couple of things more, seems like some edge cases where not accounted for by the tears within tensordict